pkg/network/ebpf/c/protocols/http2/decoding.h

```diff
@@ -488,7 +626,7 @@ int socket__http2_frames_parser(struct __sk_buff *skb) {
     local_skb_info.data_off = current_frame.offset;
 
     parse_frame(skb, &local_skb_info, &dispatcher_args_copy.tup, http2_ctx, &current_frame.frame);
-    if (local_skb_info.data_off >= skb->len) {
+    if (local_skb_info.data_off >= local_skb_info.data_end) {
```

Comment:
I believe we should consistently provide warnings about the use of `data_end` versus `skb->len`. While it may be clear in the context of this PR, I think we should include documentation explaining why we are not choosing len in certain cases.

Reply:
we should not use `skb->len`
and we're in the process of migrating the remains